types/multi-progress/index.d.ts

```diff
@@ -5,10 +5,14 @@
 
 /// <reference types="node"/>
 
-import ProgressBar, { ProgressBarOptions } from 'progress';
+import * as ProgressBar from 'progress';
```

Comment:
Would be better to use `import ProgressBar = require("progress")` to match the `export=`